### PR TITLE
Get todo list items from backend API instead of local storage

### PR DESCRIPTION
**Pull request by lis-anna**

### CODE REVIEW

**Comment by tarikutk** [comment]
I think .gitigore is empty.

**Comment by tarikutk** [comment]
now, it looks good. put .env in gitignore as well. we dont want to push credentials in github.

**Comment by tarikutk** [comment]
before you commit changes, you can also see which ones are getting commited by doing 'git status'

**Comment by lis-anna** [comment]
I've updated gitignore file, added .env

**Comment by tarikutk** [comment]
put .env in server folder as well. I am still seeing server/env in 'files changed'

**Review by tarikutk — APPROVED** [review]

**Comment by tarikutk** [comment]
get list item is working Good. and we will discuss the rest of them after we  fix the structure.

File: src/TodoList.js [hunk 1]
```diff
@@ -1,58 +1,77 @@
...
+  useEffect(() => {
+    axios
+      .get('http://localhost:3001/api/v1/todos?list=636ecac0e0a63a39c7e6217a', {
+        headers: {
+          Authorization:
+            'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2MzZlYmUzNGUwYTYzYTM5YzdlNjIxNzMiLCJ1c2VybmFtZSI6Imxpc2FubmEiLCJpYXQiOjE2NjgyMDQ0NTEsImV4cCI6MTY2ODI5MDg1MX0.yvqj2N1E_2DvuPE-5RbW59zjDAWE0SqTSY0xTGNNZ84',
+          'Content-Type': 'application/json',
+          accept: '*/*',
+        },
+      })
```
Comment (tarikutk) [inline src/TodoList.js:23]: by the way this is alsp secre. we dont want to push it to github. it is better to save in .env then reference it from there.

**Pull request merged**